src/collections/hash/map.rs

```diff
@@ -2054,17 +2054,15 @@ impl RandomState {
         // iteration order allows a form of DOS attack. To counter that we
         // increment one of the seeds on every RandomState creation, giving
         // every corresponding HashMap a different iteration order.
-        thread_local!(static KEYS: Cell<(u64, u64)> = {
```

Comment:
If we are not going to use TLS here, I'd rather keep the TLS version commented out and just use [the previous implementation](https://github.com/rust-lang/rust/pull/33318/files#diff-1ab410241f658d595f1c5eec7a979b01L1670).